src/Transports.AspNetCore.SystemTextJson/Transports.AspNetCore.SystemTextJson.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

```suggestion
<Project Sdk="Microsoft.NET.Sdk.Web">
```
so you could remove ` <FrameworkReference Include="Microsoft.AspNetCore.App" />`

---

Hmm, good idea, but it's a little weird in reality because:
1. It gives the project a web app icon in the sln explorer
2. VS then wants to auto-add a launchSettings.json to the project.

I'd be keen to leave this as is if it's ok with you?

---

ok